src/data_loader.py

```diff
@@ -0,0 +1,160 @@
+import os
+import PIL
+import torch
+import random
+import collections
+import numpy as np
+import pandas as pd
+
+
+
+class ImageSegmentaionDataset():
```

Comment:
According to this [tutorial](https://pytorch.org/tutorials/beginner/data_loading_tutorial.html), it's customary to extend one of the Dataset classes from pytorch.
For example:
```
class FaceLandmarksDataset(Dataset):
    """Face Landmarks dataset."""
```
Could you extend torch.utils.data.Dataset?

Reply:
Done